static/app/views/alerts/rules/details/metricChartOption.tsx

```diff
@@ -0,0 +1,398 @@
+import color from 'color';
```

Comment:
everything in here is just what was in metricChart.tsx pulled out so we can import it in chartcuterie